src/components/navbar/ContactNav.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import smilies from '../../assets/smilies.svg';
+import Modal from '../common/Modal';
+import ContactForm from './ContactForm';
+import { contact, closeModal } from '../../actions/navbarActions';
+import { routes } from '../../constants/routesPaths';
+import { Link } from 'react-router';
+
+class ContactNav extends Component {
+  render() {
+    return (
+      <div className="background-container">
+        <div className="top-container">
+          <Modal isOpen={true} closeModal={closeModal}>
```

Comment:
When the prop is a boolean as `isOpen` is not necessary to declare the `true` value. Just `isOpen` should work.

```javascript
<Modal isOpen closeModal={closeModal} >
```